alice_blue/alice_blue.py

```diff
@@ -570,7 +570,7 @@ def place_basket_order(self, orders):
             if i['order_type'] == OrderType.Limit:
                 if "price" not in i:
                     raise TypeError("Each element in orders should have key 'price' if its a limit order ")
-                if isinstance(i['price'], float):
+                if not isinstance(i['price'], float):
```

Comment:
This change is already done in pull request #102. If I merge that then this will be in problems.
Please segregate the changes for both issues/PRs
Like PR #102 is for issue #99 and
PR #103 is for #100 
You are doing a good job.